redirects-list.json

```diff
@@ -27,6 +27,9 @@
   "^/doc/tutorials/versioning(/.*)?$                                                      /doc/use-cases/versioning-data-and-model-files/tutorial",
   "^/doc/tutorials(/.*)?                                                                  /doc/start",
 
+  "^/doc/start/data-versioning$                                                           /doc/start/data-and-model-versioning",
+  "^/doc/start/data-access$                                                               /doc/start/data-and-model-access",
```

Comment:
Can we insert these in line 25 instead? Right after the `/doc/get-started` redirect please. No blank lines needed. Thanks

Reply:
Done in b46cca31. Thanks.